lmcache/v1/storage_backend/connector/mooncakestore_connector.py

```
@@ -234,20 +497,23 @@ async def put(self, key: CacheEngineKey, memory_obj: MemoryObj):
             )
         except asyncio.TimeoutError:
             logger.warning(
-                f"Timeout when putting key {key_str} from mooncake store."
+                f"Timeout when putting key {key_str} using put_parts. "
```

same silent error handling.